src/System.Console/tests/ManualTests/ManualTests.cs

```diff
@@ -0,0 +1,125 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System
+{
+    public class ConsoleManualTests
+    {
+        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
```

Comment:
If these only run when an environment variable is set anyways, why not add them to the existing System.Console test proj? Requiring the env variable when the tests are their own project seems like a hassle.